### DISCUSSION TITLE
An example to configure S3

### DISCUSSION THREAD

**Discussion by nnthuanegany** [saleor/saleor#15055] [Deployment]
Hi everyone,

I am configuring S3 for Saleor 3x, as below

Docs: https://docs.saleor.io/docs/3.x/setup/media-s3#serving-media-files-from-an-s3-bucket

```bash
export MEDIA_URL="https://mycdn.com/saleor/media/"
export STATIC_URL="https://mycdn.com/saleor/static/"
export AWS_ACCESS_KEY_ID="my_id"
export AWS_SECRET_ACCESS_KEY="my_key"
export AWS_DEFAULT_ACL="public-read"
export AWS_MEDIA_BUCKET_NAME="my_bucket_name"
export AWS_MEDIA_CUSTOM_DOMAIN="https://mycdn.com"
export AWS_QUERYSTRING_AUTH="True"
export AWS_QUERYSTRING_EXPIRE="30"
export AWS_STATIC_CUSTOM_DOMAIN="https://mycdn.com"
export AWS_STORAGE_BUCKET_NAME="my_bucket_name"
```

I don't know when there will be https or just domain.

And for custom domains, does it include the path (subfolder) or not?...

Please help me! and thank for all!

**Comment by jestjosh** [comment]
Did you ever figure this out? Might save me some time but w/e.

**Comment by pitkes22** [comment]
The answer is only domain, no protocol, no slash, just `example.com`... :beers:
